dctest.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+function dc {
+    ROOT_DIR=$(dirname $(pwd)) docker-compose -f tools/docker/boardfarm-ci/docker-compose.yml $@
+}
+if [ $# -eq 0 ]; then
+    dc up boardfarm
+else
+    if [ $1 == "-h" ]; then
```

Comment:
I believe shellcheck will complain here...
```suggestion
    if [ "$1" == "-h" ]; then
```